### DISCUSSION TITLE
Is There any chance to Get Reactor Extension and Adetailer Extension for SwarnUI

### DISCUSSION THREAD

**Discussion by Shahfahad7866** [mcmonkeyprojects/SwarmUI#229] [General]
Is There any chance to Get Reactor Extension and Adetailer Extension for SwarnUI. Faceswap addon is really needed for this Application.

**Comment by jwax33** [comment]
Reactor is already available.  Look in your left side options pane.  I can't answer about Adetailer, though.

**Comment by Jelosus2** [comment]
Adetailer is a built-in feature of Swarm. Might want to check this: https://github.com/mcmonkeyprojects/SwarmUI/blob/master/docs/Features/Prompt%20Syntax.md#automatic-segmentation-and-refining

**Comment by mcmonkey4eva** [comment]
ReActor is available in the FaceTools extension, see extensions list @ https://github.com/mcmonkeyprojects/SwarmUI/blob/master/docs/Extensions.md#how-to-find-extensions
